terraform/load_balancer.tf

```diff
@@ -26,4 +27,9 @@ module "lb_storage_service" {
   service_lb_security_group_ids = [
     "${local.service_lb_security_group_id}",
   ]
+
+  # We set a high timeout here and on the dashboard load balancer to allow
+  # Archivematica time to prepare large AIP files for download
+  # If nginx is sending back 499 responses, this may not be high enough.
```

Comment:
I'd duplicate this comment to the dashboard LB as well, in case the two blocks ever get separated.